src/components/DivElement.js

```diff
@@ -1,6 +1,15 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(text) {
+    super('div');
+    this.text = text;
```

Comment:
Take advantage of parent constructor and pass in text as the second argument instead of assigning it over again.